plugins/inputs/sqlserver/sqlqueriesV2.go

```diff
@@ -465,13 +465,19 @@ SET @SqlStatement = @SqlStatement + CAST(N' WHERE	(
 				''Readahead pages/sec'',
 				''Lazy writes/sec'',
 				''Checkpoint pages/sec'',
+				''Free pages'',
```

Comment:
Could you put those documentation links for the counters in the source code, maybe on the line before declaring sqlPerformanceCountersV2?  I'm hoping doc links will help out future maintainers who are not as familiar with SQL Server.  Thanks!

Reply:
> Could you put those documentation links for the counters in the source code, maybe on the line before declaring sqlPerformanceCountersV2? I'm hoping doc links will help out future maintainers who are not as familiar with SQL Server. Thanks!

@reimda Thank you for the prompt review of the changes